### PR TITLE
employees ui update

### PR DESCRIPTION
**Pull request by FacuOrellana**
Employees UI update

### CODE REVIEW

File: src/employees.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
 import * as React from "react"
 import { Datagrid, DateField, EmailField, List, TextField, ReferenceField } from 'react-admin';
-import { DateInput, Edit, SimpleForm, TextInput, Create, SelectInput, ReferenceInput } from 'react-admin';
+import { DateInput, Edit, SimpleForm, TextInput, Create, SelectInput, ReferenceInput,SelectArrayInput,ChipField,ReferenceArrayInput } from 'react-admin';
```
Comment (agustinb4) [inline src/employees.js:3]: Please format this line

File: src/employees.js [hunk 2]
```diff
@@ -28,42 +25,53 @@ export const EmployeeList = () => (
 export const EmployeeEdit = () => (
     <Edit>
         <SimpleForm>
-            <ReferenceInput source="companyId" reference="companies">
-                <SelectInput source="name" />
-            </ReferenceInput>
-            <TextInput disabled source="id" />
-            <TextInput source="internalId" />
+        <TextInput source="internalId" />
```
Comment (agustinb4) [inline src/employees.js:28]: Check the format here

File: src/employees.js [hunk 3]
```diff
@@ -28,42 +25,53 @@ export const EmployeeList = () => (
...
+            <TextInput source="emergencyContactFullName" />
+            <TextInput source="emergencyContactPhone" />
+            <ReferenceArrayInput reference="roles" source="profile">
+            <SelectArrayInput>
+                 <ChipField source="profiles" />
+            </SelectArrayInput>          
+            </ReferenceArrayInput>
         </SimpleForm>
     </Create>
 );
```
Comment (agustinb4) [inline src/employees.js:77]: Add a new line at the end of the file here

**Review by agustinb4 — CHANGES_REQUESTED** [review]
Some minor changes in the style

**Review by agustinb4 — APPROVED** [review]

**Review by reconeazaretzky — APPROVED** [review]

**Pull request merged**